src/services/language.ts

```diff
@@ -140,7 +140,8 @@ export class LanguageService {
 
   convertDate(date){
     if(this._lang === 'fa'){
-      momentJalali.loadPersian(true);
+      // momentJalali.loadPersian(true);
+      momentJalali.loadPersian();
```

Comment:
It is strange. It did not work with 'true' parameter for me. For uploading on ionic server I should remove the 'true' parameter.